File: JavaScript/01.JavaScript-Basic-February-2021/14.exams/2019.05.02_03-Exam/1.foodDelivery.js
function food(input) {
    let chicken = Number(input[0]);
    let fish = Number(input[1]);
    let vegetarian = Number(input[2]);

    let chickenPrice = 10.35;
    let fishPrice = 12.40;
    let vegetarianPrice = 8.15;

    let total = 0;
    let menu = 0;

    menu = chicken * chickenPrice + fish * fishPrice + vegetarian * vegetarianPrice;
    total = menu + menu* 0.2 + 2.50;

    console.log(`Total: ${total.toFixed(2)}`);
}

food(["2", "4", "3"]);
food(["10", "0", "6"]);

File: Python/01.Python-Basic-April-2019/15.exams/exam_2019.05.02/1.food_delivery.py
chicken = int(input())
fish = int(input())
veg = int(input())

all_menu = chicken * 10.35 + fish * 12.40 + veg * 8.15
desert = all_menu * 0.2
delivery = 2.5

total = all_menu + desert + delivery

print(f'Total: {total:.2f}')